eng/tox/tox.ini

```diff
@@ -118,6 +119,40 @@ commands =
     {envbindir}/python {toxinidir}/../../../eng/tox/run_mypy.py -t {toxinidir}
 
 
+[testenv:pyright]
+skipsdist = true
+skip_install = true
+usedevelop = true
+changedir = {toxinidir}
+setenv =
+  {[testenv]setenv}
+  PROXY_URL=http://localhost:5003
```

Comment:
```suggestion
  PROXY_URL=http://localhost:5018
```

Reply:
No common PROXY_URLs across the tox environments. Don't want them to spin up proxy instances and stomp on each other 👍